src/transaction_builder.js

```diff
@@ -232,6 +232,11 @@ function prepareInput (input, ourPubKey, redeemScript, witnessValue, witnessScri
       expanded.signatures = input.signatures
     }
 
+    let signScript = witnessScript
+    if (expanded.type === SCRIPT_TYPES.P2WPKH) {
+      signScript = payments.p2pkh({ pubkey: expanded.pubkeys[0] }).output
+    }
```

Comment:
I thought this would resolve the signing issues for `P2WSH(P2WPKH)`... but, it still fails.  Investigating.